addons/refuel/functions/fnc_getFuel.sqf

```diff
@@ -19,11 +19,4 @@ params [["_source", objNull, [objNull]]];
 
 if (isNull _source) exitWith {0};
 
-private _fuel = _source getVariable QGVAR(currentFuelCargo);
-
-if (isNil "_fuel") then {
-    _fuel = getNumber (configOf _source >> QGVAR(fuelCargo));
-    _source setVariable [QGVAR(currentFuelCargo), _fuel, true];
-};
-
-_fuel
+_source getVariable [QGVAR(currentFuelCargo), 0];
```

Comment:
This change is unnecessary and breaks public API.

Reply:
Okay, I'll revert this